### PR TITLE
US_10_Tsk_81: Update discussion pane based on the feedback type

### PR DESCRIPTION
**Pull request by ishansarangi**

### CODE REVIEW

File: ui/src/Components/FeedbackType.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const FeedbackType = {
+  My: 1,
+  Employee: 2,
```
Comment (mohantysabyasachi27) [inline ui/src/Components/FeedbackType.js]: what is 'MY'? Change the names to something meaningful

File: ui/src/Components/ManagerPane.js [hunk 2]
```diff
@@ -50,7 +57,7 @@ const ManagerPane = props => {
       </nav>
       <div className="child-content">
         {/* need to remove feedback and update using props */}
-        <Feedback />
+        {FeedbackView(selectedIndex)}
```
Comment (mohantysabyasachi27) [inline ui/src/Components/ManagerPane.js]: function names must always be in lowercase

File: ui/src/Components/FeedbackType.js (hunk 1 above)
Comment (ishansarangi) [reply ui/src/Components/FeedbackType.js]: done

File: ui/src/Components/ManagerPane.js (hunk 2 above)
Comment (ishansarangi) [reply ui/src/Components/ManagerPane.js]: done

**Pull request merged**